<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QDialog" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlWindow</string>
  </property>
  <widget class="QCheckBox" name="spikemap_window_cb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spike Map</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="raster_window_cb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rasters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="waveform_window_cb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Waveforms</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="absolute_threshold_radioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rms_threshold_radioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RMS Multiple</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Display:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spike Detection:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rms_threshold_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="absolute_threshold_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SpikeGLX Connection:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="connection_status_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SpikeGLX not connected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveDefaults_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreDefaults_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Defaults</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
